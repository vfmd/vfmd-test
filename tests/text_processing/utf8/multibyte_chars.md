Two-byte char:

ABÇD

ABÇ

Three-byte char:

ABḈD

ABḈ

Four-byte char:

AB𝜍D

AB𝜍

