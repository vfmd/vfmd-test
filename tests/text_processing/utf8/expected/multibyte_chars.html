<p>Two-byte char:</p>

<p>ABÇD</p>

<p>ABÇ</p>

<p>Three-byte char:</p>

<p>ABḈD</p>

<p>ABḈ</p>

<p>Four-byte char:</p>

<p>AB𝜍D</p>

<p>AB𝜍</p>

